lib/elixir_internal_certificate_web/controllers/api/v1/user_search_controller.ex

```diff
@@ -27,6 +28,32 @@ defmodule ElixirInternalCertificateWeb.Api.V1.UserSearchController do
     end
   end
 
+  def show(
+        %{
+          private:
+            %{guardian_default_claims: %{"sub" => user_id} = _guardian_default_claims} = _private
+        } = conn,
+        %{"id" => id} = _params
+      ) do
+    case Scrapers.get_user_search_by_user_id_and_id(user_id, id) do
```

Comment:
can we use `get_user_search_by_user_id_and_id!` it will raise an Ecto.NotFound exception`

Reply:
I rather keep the current implementation not edit the Phoenix's view error_view